.eslintrc.json

```diff
@@ -0,0 +1,111 @@
+{
+  "env": {
+    "mocha": true,
+    "node": true
+  },
+  "globals": {
+    "angular": "readonly",
+    "assert": "readonly",
+    "browser": "readonly",
+    "by": "readonly",
+    "element": "readonly",
+    "expect": "readonly",
+    "inject": "readonly"
+  },
+  "rules": {
+    "no-bitwise": 2,
+    "curly": 2,
+    "eqeqeq": 2,
+    "keyword-spacing": [
+      2,
+      {}
+    ],
+    "no-extend-native": 2,
+    "wrap-iife": [
+      2,
+      "any"
+    ],
+    "no-use-before-define": [
+      2,
+      {
+        "functions": false
+      }
+    ],
+    "space-before-function-paren": [
+      2,
+      {
+        "anonymous": "ignore",
+        "named": "never"
+      }
+    ],
+    "one-var": [
+      2,
+      "never"
+    ],
+    "array-bracket-spacing": [
+      2,
+      "never"
+    ],
+    "quote-props": [
+      2,
+      "as-needed",
+      {
+        "keywords": true
+      }
+    ],
+    "key-spacing": [
+      2,
+      {
+        "beforeColon": false,
+        "afterColon": true
+      }
+    ],
+    "comma-style": [
+      2,
+      "last"
+    ],
+    "space-unary-ops": [
+      2,
+      {
+        "words": false,
+        "nonwords": false
+      }
+    ],
+    "space-infix-ops": 2,
+    "quotes": [
```

Comment:
polyjuice converted `validateQuoteMarks` to the `quotes` eslint rule via https://github.com/brenolf/polyjuice/blob/master/lib/dictionaries/jscs/validateQuoteMarks.js. I assumed that the code style preferred double quotes and Grunt required for `Gruntfile.js` so I ran the eslint fixer. I assumed wrong so I'll revert the code reformat and remove this rule.